teamhanko/hanko
# Fix: README.md

Added Twitter and Discord badges to the text to provide quick access to the Hanko community. The badges will enhance user experience and enable easy navigation to the community platforms.



# Description

Added Twitter and Discord badges to the text to provide quick access to the Hanko community. The badges will enhance user experience and enable easy navigation to the community platforms.

# Implementation

The badges were added directly to the existing text, utilizing Markdown syntax. The purpose is to make the community platforms easily accessible with a single click.

# Tests

To verify the changes, follow these steps:

1. Open the relevant page where the Hanko community text is located.
2. Verify that the Twitter and Discord badges are visible and clickable.
3. Confirm that clicking the badges redirects users to the respective Hanko community pages.

# Todos

None.

# Additional context

The addition of the badges aims to create an eye-catching and user-friendly experience, facilitating a seamless transition to the Hanko community platforms with just one click.
![Screenshot from 2023-10-31 00-06-02](https://github.com/teamhanko/hanko/assets/98478305/d328ba03-a94e-40a3-8638-d791d66895c0)
![Screenshot from 2023-10-31 00-06-25](https://github.com/teamhanko/hanko/assets/98478305/3bb200bb-80a5-4d8a-a94e-f2d3c55e76f1)
